rdkit-stubs/Chem/rdChemReactions.pyi

```diff
@@ -92,7 +94,9 @@ class ChemicalReaction:
     @classmethod
     def RunReactantInPlace(cls, *args, **kwargs) -> Any: ...
     @classmethod
-    def RunReactants(cls, RDKit, boost) -> Any: ...
+    def RunReactants(
+        cls, reactants: Sequence[Mol], max_products=1000
```

Comment:
Is this really a class method?

Reply:
Same question applies to some other instances I see of this pattern in this pr.

Reply:
I neglected to remove the classmethods, will do now.